File: Day26/sum_of_positive_numbers.py
"""Function to find the sum of positive number in a array."""
def get_sum_of_positive_numbers(arr):
    """ initialize total_sum variable with 0"""
    total_sum =0
    for num in arr:
        if num>0:
            total_sum += num

    return total_sum

#Let's Start here
arr1 = [-8,9,4,-3,5,-4,2]
ANS= get_sum_of_positive_numbers(arr1)
print("Sum of Positive number: ",ANS)

File: Day26/SumOfPositiveNumbers.java
public class SumOfPositiveNumbers {
    //Function to find the sum of positive number in a array.
    public static int getSumOfPositiveNumbers(int arr[]){

        int sum =0;
        for(int i=0;i<arr.length;i++){
            if(arr[i]>0){
                sum += arr[i];
            }
        }
        return sum; 
    }
    public static void main(String[] args) {
        int arr[]={-8,9,4,-3,5,-4,2};
        int result = getSumOfPositiveNumbers(arr);
        System.out.println("Sum of Positive number: " + result);
    }
}
